### PR TITLE
add customize baseUrl and confirmation flag

### PR DESCRIPTION
**Pull request by Hector68**

### CODE REVIEW

File: src/assets/src/js/sortable-column.js [hunk 1]
```diff
@@ -66,34 +67,43 @@
...
         },
 
-        move: function(action, additionalParams) {
+        getBaseUrl: function() {
+            return this.getSortable().sortable('option', 'baseUrl');
+        },
+
+        needConfirmationOnMove: function () {
+            return this.getSortable().sortable('option', 'needConfirmationOnMove');
+        },
```
Comment (arogachev) [inline src/assets/src/js/sortable-column.js:92]: Linebreak is missing here.

File: src/assets/src/js/sortable-column.js [hunk 2]
```diff
@@ -66,34 +67,43 @@
...
             var row = this;
 
             if (!this.isSortable()) {
                 return;
             }
 
-            if (!confirm(this.getMoveConfirmationText())) {
-                this.resetPosition();
-                this.getSortable().sortable('cancel');
+            if (this.needConfirmationOnMove === true) {
```
Comment (arogachev) [inline src/assets/src/js/sortable-column.js]: Nested ifs can be refactored to:

``` js
if (this.needConfirmationOnMove === true && !confirm(this.getMoveConfirmationText()) {
```

File: src/grid/SortableColumn.php [hunk 3]
```diff
@@ -39,6 +39,16 @@ class SortableColumn extends Column
 
 
     /**
+     * @var bool
+     */
+    public $needConfirmationOnMove = true;
```
Comment (arogachev) [inline src/grid/SortableColumn.php]: Can be simplified to just `confirmMove`.

File: src/assets/src/js/sortable-column.js [hunk 4]
```diff
@@ -66,34 +67,43 @@
...
+        getMoveConfirmationText: function () {
             return this.getSortable().sortable('option', 'moveConfirmationText');
         },
 
-        move: function(action, additionalParams) {
+        getBaseUrl: function() {
+            return this.getSortable().sortable('option', 'baseUrl');
+        },
+
+        needConfirmationOnMove: function () {
```
Comment (arogachev) [inline src/assets/src/js/sortable-column.js]: Rename to `isMoveConfirmed`.

**Comment by Hector68** [comment]
done

**Comment by arogachev** [comment]
Thanks! But I think it's not good idea to mix new feature implementation and code style fixes in the same PR.

Could you please revert it back to `function() {` for functions and `{ property: value }` for one-liner objects with one value? And I will merge it and add info to the docs.

**Comment by arogachev** [comment]
If you are busy, I can do it in separate commit, but a bit later.

**Comment by arogachev** [comment]
> Hector68 wants to merge 2 commits into arogachev:master from Hector68:master 

Please create separate branch for PR.

**Comment by Hector68** [comment]
I has created separate branch and new set new PR.

**Comment by arogachev** [comment]
Merged related PR: https://github.com/arogachev/yii2-sortable/pull/26

**Pull request closed without merging**